authorizer-app-backend/src/main/java/org/radarbase/authorizer/doa/RestSourceUserRepositoryImpl.kt

```diff
@@ -140,6 +140,26 @@ class RestSourceUserRepositoryImpl(
         }
     }
 
+    override fun queryAllWithElapsedEndDate(
+        sourceType: String?
+    ): List<RestSourceUser> {
+        var queryString = "SELECT u FROM RestSourceUser u WHERE"
+        queryString += " u.endDate < CURRENT_TIMESTAMP"
```

Comment:
By merging the string, the editor has more context on what properties are being queried:
```suggestion
        var queryString = "SELECT u FROM RestSourceUser u WHERE u.endDate < CURRENT_TIMESTAMP"
```
Alternatively, you can use Kotlin's multiline string to help with readability:
```kotlin
        var queryString = """
               SELECT u
               FROM RestSourceUser u
               WHERE u.endDate < CURRENT_TIMESTAMP
        """.trimIndent()
```